### DISCUSSION TITLE
Forward DNS

### DISCUSSION THREAD

**Discussion by iambabyninja** [XTLS/Xray-core#2893] [Q&A] [answered]
![image](https://github.com/XTLS/Xray-core/assets/104270279/9cc483ca-d7d9-4ced-be6d-5613d1b173e5)
亲爱的朋友们！
希望这条信息找到你们时，你们身体健康。
我想询问，在我们使用REALITY的情况下，为我们的IP地址设置FORWARD DNS（PTR），对于我们作为伪装使用的网站，有没有意义？

My dear friends!
I hope this message finds you in good health.
I wanted to inquire, in the case when we use REALITY, is there any sense in setting up FORWARD DNS (PTR) for our IP address, for the site we are using as camouflage?

**Comment by chise0713** [comment, marked as answer]
可以尝试，但是我认为意义不大，因为部分厂商的域名通常是解析到自家的CDN上，而CDN的rDNS一般不会是你“偷”的那个域名的rDNS
